packages/nouns-contracts/contracts/libs/Inflate.sol

```diff
@@ -0,0 +1,924 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity >=0.8.0 <0.9.0;
+
+/// @notice Based on https://github.com/madler/zlib/blob/master/contrib/puff
```

Comment:
@eladmallel let's mention in a `/// @dev` comment that the diff from the original is for gas optimization